src/components/Book.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+function Book(props) {
+  const { title, author } = props;
+
```

Comment:
### [OPTIONAL]

- [ ] It will be better and cleaner to destructure the props as the parameters to the book function. This will make your code even cleaner. example: ⬇️ 

```
function Book ({title, author}) {
}